blocks/market-commentary/market-commentary.js

```diff
@@ -25,10 +25,10 @@ function createMarketCommentaryCard(cardData, placeholders) {
         div(
           { class: 'info' },
           p(
-            span(`${placeholders.powerby}`),
+            span(`${blockConfig.powerby}`),
```

Comment:
@ravverma this is repeated in many blocks , IMO we should take it from placeholder

Reply:
I though it was for this only but I have checked it is using several places. reverting this change.